cabal.project

```diff
@@ -0,0 +1,37 @@
+packages:
+    api
+  , sdk
+  , otlp
+  , exporters/handle
+  , exporters/in-memory
+  , exporters/otlp
+  , propagators/w3c
+  , instrumentation/conduit
+  , instrumentation/cloudflare
+  , instrumentation/http-client
+  , instrumentation/persistent
+  , instrumentation/postgresql-simple
+  , instrumentation/yesod
+  , instrumentation/wai
+
+source-repository-package
+    type: git
+    location: https://github.com/ysangkok/proto-lens
```

Comment:
Is ysangkok/proto-lens a private repo?

Reply:
I think that the repo was deleted when the associated PR was merged into `master`

Reply:
Thanks! I see. Maybe this https://github.com/google/proto-lens/pull/440?